Resources/Locale/en-US/datasets/figurines.ftl

```diff
@@ -0,0 +1,180 @@
+figurines-hop-1 = Papers, please.
+figurines-hop-2 = You are fired.
+
+figurines-passenger-1 = How to change hands?
+figurines-passenger-2 = Call the evac.
+
+figurines-greytider-1 = Man, this party stinks. I fucking hate these people.
+figurines-greytider-2 = Uh-oh, who's lost their stunbaton?
+figurines-greytider-3 = Robust.
+figurines-greytider-4 = I'm not me without a toolbox.
+figurines-greytider-5 = Grey tide station wide!
+
+figurines-clown-1 = Honk!
+figurines-clown-2 = Banana!
+figurines-clown-3 = Soap!
+figurines-clown-4 = HoP has one clown, HoS has the whole department.
+figurines-clown-5 = Do I annoy you?
+
+figurines-holoclown-1 = I'm helping my older brother.
+
+figurines-mime-1 = ...
+figurines-mime-2 = ...
+figurines-mime-3 = ....
+figurines-mime-4 = .......
+figurines-mime-5 = ................
+
+figurines-musician-1 = Never gonna give you up!
+figurines-musician-2 = Never gonna let you down!
+
+# figurines-boxer-
+
+figurines-captain-1 = Crew, the Nuke Disk is safely up my ass.
+figurines-captain-2 = Glory to NT!
+
+figurines-hos-1 = Space law? What?
+figurines-hos-2 = Shot the clown.
+
+figurines-warden-1 = Execute him for breaking in!
+
+figurines-detective-1 = The butler did it.
+
+figurines-security-1 = I am the law!
+figurines-security-2 = You have violated article 1984.
+
+figurines-lawyer-1 = Better Call Saul!
+figurines-lawyer-2 = Objection!
+
+figurines-cargotech-1 = DRAGON ON ATS!
+figurines-cargotech-2 = I sold the station!
+figurines-cargotech-3 = Brain bounty? I don't have a brain.
+
+figurines-salvage-1 = Megafauna? It was mega easy.
+
+figurines-qm-1 = Who stole the shuttle?
+figurines-qm-2 = I won't approve the guns.
+figurines-qm-3 = I didn't buy those guns!
+figurines-qm-4 = One toys crate for ma fellow clown!
+figurines-qm-5 = Time to spent all money on gambling.
+figurines-qm-6 = Viva La Cargonia!
+figurines-qm-7 = Fill the form.
+
+figurines-ce-1 = Everyone to the briefing!
+figurines-ce-2 = Wire the solars!
+figurines-ce-4 = How to setup the TEG?
+figurines-ce-5 = SINGULOOSE!
+figurines-ce-6 = TESLALOOSE!
```

Comment:
According to my highly accurate opinion poll (i.e. used the search function on discord) it should be `TESLOOSE!`
![grafik](https://github.com/user-attachments/assets/145a3c0c-2f5e-4005-a250-f8989e329939)
![grafik](https://github.com/user-attachments/assets/7c097ed1-a60a-439f-9822-70bc5b65cf45)